.github/workflows/ci-conda.yml

```diff
@@ -19,6 +22,12 @@ jobs:
     name: Conda
     runs-on: ${{ matrix.os }}
     
+    if: |
+      github.event.action != 'labeled' ||
```

Comment:
Could you add a comment that explains what this condition is intended to do?

Reply:
Done.

Reply:
Thanks, but why should it NOT run when the action is "labeled"?

Reply:
When the action is "labeled" it should only run when one of the specified labels is added. So in this case this resolves to `false || true`.

Reply:
Oh, now I see. Thanks.